code/src/components/TodoItem.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react'
+
+const TodoItem = ({ decription, checkComplete, onChangeToggle }) => {
```

Comment:
I remember you saying that you had trouble spelling description right 😅 but at least it's consistent and working! 😉